internal/scaffold/olm-catalog/testdata/deploy/olm-catalog/app-operator/0.1.0/app-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -104,6 +107,14 @@ spec:
     type: MultiNamespace
   - supported: true
     type: AllNamespace
+  keywords:
+  - Placeholder Keyword
+  maintainers:
+  - email: Placeholder email
+    name: Placeholder name
   maturity: alpha
-  provider: {}
```

Comment:
@estroz look how it was before. In POV is more clear for the user to check how to fill it by seen he PLACEHOLDER fields